com.github.rizinorg.Cutter.yml

```diff
@@ -0,0 +1,29 @@
+#flatpak-builder --force-clean --user --install ./build ./com.github.rizinorg.cutter.yml
+app-id: com.github.rizinorg.Cutter
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: cutter
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+modules:
+  - name: cutter
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
this won't work.
There is no network access during builds.

Reply:
Yeah I guessed it loking at the build logs Haha. The last version does not need it.